### PR TITLE
Replace Korok labels with Korok IDs

### PR DESCRIPTION
**Pull request by savage13**
Korok labels were replaced with Korok IDs from the Korok ID Map.  Colors and styles from the original map were used.  

Clicking on a Korok will bring up its info using `AppMapDetailsObj`, this requires knowing the `objid` ahead of time. Instead we search for the Korok using it `map_name`, `map_type` and `hash_id`.

These changes required modifications to `public/game_files/map_summary/MainField/static.json` with the Korok ID, `map_name`, `map_type`, `map_static`, and `hash_id`. This file included details on the Koroks  that were removed. The size of `static.json` decreased from 790K to 310K.

This does not include changes the `objs` database, but those are not required.

See Issue #55

---
This change is [Reviewable](https://reviewable.io/reviews/zeldamods/objmap/56)

### CODE REVIEW

**Comment by savage13** [comment]
It looks like the `Static.json` and `Location.json` files needed by `tools/ map_gen_markers.py` to generate `static.json` are located in the radar repo.  I will look into modifying `tools/ map_gen_markers.py` to create a new `static.json` file.

**Comment by savage13** [comment]
Small modification of `tools/map_gen_markers.py` to include Korok IDs that are stored in korok_ids.json in the root directory (not sure where else to put it).

File: src/MapMarker.ts [hunk 1]
```diff
@@ -209,14 +216,45 @@ const KOROK_ICON = (() => {
...
+      className: class_to_color(id),
     });
     this.info = info;
+    // @ts-ignore
+    this.obj = info;
+  }
+}
+
+// Convert first letter of Korok ID to CSS classname
+function class_to_color(id: string): string {
```
Comment (leoetlino) [inline src/MapMarker.ts:235]: classToColor (for consistency with the other function names)

File: src/components/AppMap.ts [hunk 2]
```diff
@@ -310,6 +312,26 @@ export default class AppMap extends mixins(MixinUtil) {
...
+      const component = MARKER_COMPONENTS[type];
+      const group = new MapMarkerGroup(
+        markers.map((m: any) => new (component.cl)(this.map, m, { showLabel: this.showKorokIDs })),
+        valueOrDefault(component.preloadPad, 1.0),
+        valueOrDefault(component.enableUpdates, true));
+      this.markerGroups.set(type, group);
+      group.addToMap(this.map.m);
+      group.update();
+    }
+  }
```
Comment (leoetlino) [inline src/components/AppMap.ts:334]: newline after this line

**Review by leoetlino — APPROVED** [review]

**Pull request merged**